php/auth.php

```diff
@@ -0,0 +1,49 @@
+<?php
+$ERRORLOG = getenv('PHP_ERROR_LOG');
```

Comment:
Does this variable need to be set? PHP is yelling that the file name on line 5 is empty, which I assume would mean that this is empty.